packages/rpcs/cfx_epochNumber/index.test.js

```diff
@@ -29,37 +29,37 @@ describe('@cfxjs/cfx_epoch-number', function () {
       expect(input.f).toHaveBeenCalledTimes(1)
     })
 
-    it('should call the injected fetch function with the default params', async function () {
-      let input = {
-        f: jest.fn(p => p),
-      }
-      let res = await main(input)
-      expect(res).toStrictEqual({params: ['latest_state']})
-      expect(input.f).toHaveBeenCalledTimes(1)
+    // it('should call the injected fetch function with the default params', async function () {
```

Comment:
why the test be commented

Reply:
I've removed these test cases.

`params: null` is invalid json rpc request. it will be rejected at https://github.com/Conflux-Chain/helios/blob/412b51fcd330d432c35dab27a39ea07fb2325e8a/packages/rpc-engine/middlewares/validate-json-rpc-format.js#L12
`params: [null]` is valid and the `null` will be replaced with `latest_state` or `latest` depends on the network type at https://github.com/Conflux-Chain/helios/blob/412b51fcd330d432c35dab27a39ea07fb2325e8a/packages/rpc-engine/middlewares/validate-rpc-data.js#L34